sass/helpers/base.sass

```diff
@@ -0,0 +1,38 @@
+// Base
+
+*
+  box-sizing: border-box
+  margin: 0
+  padding: 0
+
+html
+  font-size: 62.5%
+  background-image: url("../assets/bg.jpg")
+  background-repeat: no-repeat
+  background-size: cover
+
+body
```

Comment:
Element selectors are definitely OK for inheritable properties such as font: family, size, color, etc